pxr/imaging/hio/glslfxConfig.cpp

```diff
@@ -310,7 +310,7 @@ HioGlslfxConfig::_GetSourceKeyMap(VtDictionary const & dict,
 
     const VtDictionary& specDict = techniqueSpec.UncheckedGet<VtDictionary>();
     // get all of the shader stages specified in the spec
-    for (const std::pair<std::string, VtValue>& p : specDict) {
+    for (const std::pair<std::string, VtValue> p : specDict) {
```

Comment:
You should be able to avoid the copy by making `p` `const auto&` or `const std::pair<const std::string, VtValue>&`.